main.py

```diff
@@ -69,5 +69,5 @@ def main():
     except Exception as e:
         logging.exception(e)
         error_message = traceback.format_exc()
-    if os.getenv("ENABLE_MAILER"):
+    if int(os.getenv("ENABLE_MAILER", default=0)):
```

Comment:
Is this a fix that needs to go into #4?

Reply:
@zkagin Yes that change probably belonged there instead. Caught the issue while testing this branch.